frameworks/Clojure/pedestal/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk-amd64
```

Comment:
Per the changes in #1512, replace this line with `source $IROOT/java7.installed`